.tekton/listeners/node-pre-release.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: triggers.tekton.dev/v1beta1
+kind: TriggerBinding
+metadata:
+  name: node-prerelease-binding
+spec:
+  envFrom:
+    - configMapRef:
+        name: environment-properties
+  params:
+    - name: branch
+      value: $(params.target-branch)
+    - name: target-branch
+      value: $(params.target-branch)
+    - name: type
+      value: "node-prerelease"
+    - name: repository
+      value: $(params.repository-url)
+    - name: commit-id
+      value: $(param.commit-id)
+      default: ""
+    - name: node-version
+      value: $(params.node-version)
+      default: ""
+    - name: npm-version
+      value: $(params.npm-version)
+      default: "" 
+    - name: pl-name
+      value: "node-pre-release"
+    - name: pipeline-ref
+      value: "default-pipeline"
+    - name: skip-cache
+      value: $(params.skip-cache)   
+      default: ""
+    - name: prerelease
+      value: "true"
+    - name: prerelease-node-version
```

Comment:
suggestion: Raise a card to make this dynamic. Otherwise we have to maintain the value of "prerelease-node-version" in the pipeline configuration. We can pull the latest available nightly/rc Node.js prerelease version.

Reply:
created one [INSTA-16692](https://jsw.ibm.com/browse/INSTA-16692)